### PR TITLE
Adding .elasticbeanstalk/config.yml :trumpet: :horse:

### PR DESCRIPTION
**Pull request by jdrager**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -54,4 +54,60 @@ jobs:
 
       - store_artifacts:
           path: test-reports
-          destination: test-reports
\ No newline at end of file
+          destination: test-reports
+  test:
+    working_directory: ~/mnl-api
+
+    docker:
```
Comment (ryangallen) [inline .circleci/config.yml]: any way to DRY this up since it's the same for the build job?

File: .elasticbeanstalk/config.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+branch-defaults:
+  master:
+    environment: mnl-api
+global:
+  application_name: mnl
+  default_ec2_keyname: XXXXXX
+  default_platform: 64bit Debian jessie v2.8.4 running Python 3.4 (Preconfigured - Docker)
```
Comment (ryangallen) [inline .elasticbeanstalk/config.yml]: Can't remember off the top of my head if this matches but is there a way to make sure we keep this and the django/Dockerfile in sync somehow?

Comment (jdrager) [reply .elasticbeanstalk/config.yml]: in sync how?  i have a hard time seeing how they're similar

File: .circleci/config.yml (hunk 1 above)
Comment (jdrager) [reply .circleci/config.yml]: great point, i'll work on figuring this out

File: .elasticbeanstalk/config.yml (hunk 2 above)
Comment (ryangallen) [reply .elasticbeanstalk/config.yml]: I'm not sure it matters, not exactly sure how you're setting up the prod environment.

I was thinking about how the django service runs on python:3.6-alpine https://github.com/monday-night-lights/mnl-api/blob/master/django/Dockerfile#L1

Comment (ryangallen) [reply .elasticbeanstalk/config.yml]: Will we use Docker in prod? Could we bundle everything into an image? 

We wouldn't need Python on the server at all at that point.

Comment (jdrager) [reply .elasticbeanstalk/config.yml]: Ahh. I thought you meant automatically in-sync, not having the same docker image.  I think I misunderstood you.  I'll look into it

**Comment by jdrager** [comment]
I'm going to merge (and squash this)

My current theory is that it is not succeeding because it is in a PR and not on Main.

**Pull request merged**